cmd/immudb/immudb.go

```diff
@@ -140,6 +146,8 @@ func setupFlags(cmd *cobra.Command, options server.Options, mtlsOptions server.M
 	cmd.Flags().StringP("address", "a", options.Address, "bind address")
 	cmd.Flags().StringP("dbname", "n", options.DbName, "db name")
 	cmd.Flags().StringVar(&o.CfgFn, "config", "", "config file (default path are configs or $HOME. Default filename is immudb.ini)")
+	cmd.Flags().Bool("eventual-consistency", options.EventualConsistency, "enable/disable eventual consistency")
+	cmd.Flags().Uint64("cache-size", options.CacheSize, "enable/disable eventual consistency")
```

Comment:
@mmeloni description here looks the same as for the enable/disable config item, i suppose it should be something like _maxcache size in MB_